testbenches/simlibs.mk

```diff
@@ -163,6 +165,19 @@ VDEFINES   += RND_ECC_WEIGHT=0
 endif
 
 
+ifeq ($(CL_MANYCORE_MEM_CFG),e_infinite_mem)
+VDEFINES   += AXI_MEMORY_MODEL=1
+VDEFINES   += ECC_DIRECT_EN
+VDEFINES   += RND_ECC_EN
+VDEFINES   += ECC_ADDR_LO=0
+VDEFINES   += ECC_ADDR_HI=0
+VDEFINES   += RND_ECC_WEIGHT=0
+endif
+
+ifeq ($(EXTRA_TURBO), 1)
+VLOGAN_VFLAGS += -undef_vcs_macro
+endif
+
```

Comment:
This was removed in the latest makefile refactor

Reply:
must have slipped back in there - removed.